state/mongodb/mongodb.go

```diff
@@ -225,7 +225,14 @@ func (m *MongoDB) Get(req *state.GetRequest) (*state.GetResponse, error) {
 	case string:
 		data = []byte(obj)
 	case primitive.D:
-		if data, err = bson.MarshalExtJSON(obj, true, true); err != nil {
+		// Setting canonical to `false`.
```

Comment:
This seems to be a breaking change. If this component is Alpha, it is OK.

Reply:
It is not alpha ... It is stable component since v1.0 ...

Reply:
> This seems to be a breaking change. If this component is Alpha, it is OK.

This is not a breaking change, but a bug fix.